pkg/nfd-master/node-updater-pool.go

```diff
@@ -41,23 +42,29 @@ func newNodeUpdaterPool(nfdMaster *nfdMaster) *nodeUpdaterPool {
 }
 
 func (u *nodeUpdaterPool) processNodeUpdateRequest(queue workqueue.RateLimitingInterface) bool {
-	nodeName, quit := queue.Get()
+	n, quit := queue.Get()
+	nodeName := n.(string)
 	if quit {
 		return false
 	}
```

Comment:
I think string casting should be moved after the quit check because according to the queue package:
```go
if len(q.queue) == 0 {
	// We must be shutting down.
	return nil, true
}
```
We risk having nil casting error
```suggestion
	if quit {
		return false
	}
	nodeName := n.(string)
```

Reply:
Yes, this is fixed in #1595. I'll rebase after that gets in